components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -1947,7 +2020,7 @@ else if (unit.equals("m")) {
             if (ms.sizeZ == 1) {
               ms.sizeZ = len;
               bytesPerAxis.put(nBytes, "Z");
-              physicalSizeZ = (physicalLen * len) / (len - 1);
+              physicalSizeZ = physicalLen;
```

Comment:
Note this change should result in the same value of the physical size Z but will affect all the physical sizes in X and Y which should force us to regenerate the configuration files for the automated testing. @melissalinkert: in https://github.com/openmicroscopy/bioformats/commit/a99279ff4c0c8bb8519b4a938168ec2742a28ebc, was there a rational for correcting `physicalLen` for the Z dimension but not the X and Y?